test/e2e/eventtype_test.go

```diff
@@ -37,6 +38,10 @@ const (
 )
 
 func TestEventtype(t *testing.T) {
+	//FIXME: enable after Eventing v1.11 is out
```

Comment:
How can we ensure that we don't forget this ?

Reply:
Yep, it's a bit tedious. But the release is this week, I'll try not to foget to remove it immediately.